### PR TITLE
chore(build): force latest stable of yarn in ci to fix Buffer warnings

### PR DESCRIPTION
**Pull request by ryanseddon**
## Description

This changes which version of yarn is installed by travis so we can avoid the `Buffer()` deprecation warnings messing up our logs.

## Detail

See yarn issue: yarnpkg/yarn#5477
See travis issue: travis-ci/travis-ci#7471

All we do is run the install shell script to grab the latest stable (1.12.3)

### CODE REVIEW

**Comment by ryanseddon** [comment]
Should we also consider adding a yarn binary and forcing the same version on local dev using `.yarnrc` `yarn-path`?

https://yarnpkg.com/lang/en/docs/yarnrc/#toc-yarn-path

**Comment by jzempel** [comment]
@ryanseddon let's not add that maintenance overhead if it's not necessary.

**Comment by austin94** [comment]
> Should we also consider adding a yarn binary and forcing the same version on local dev using .yarnrc yarn-path?

I'm leaning toward not adding in the binary unless it's necessary.

File: .travis.yml [hunk 1]
```diff
@@ -9,6 +9,14 @@ cache:
   directories:
     - node-modules
 
+env:
+  global:
+    - YARN_VERSION="1.12.3"
+
+before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version $YARN_VERSION
```
[the remark this answers is not in the document]
Comment (ryanseddon) [reply .travis.yml]: yeah so I looked at doing that and their docs need the version set I'm not sure if this is a requirement and I don't think we'd really need to update this very often at all.

[the remark this answers is not in the document]
Comment (jzempel) [reply .travis.yml]: According to this `--version` is optional: https://yarnpkg.com/lang/en/docs/install/#alternatives-stable

[the remark this answers is not in the document]
Comment (ryanseddon) [reply .travis.yml]: touche I'll remove that version bit

**Pull request merged**